### ISSUE TITLE
Move To PROD | APF | Superset Report Changes | Child linelist (AWC)

### ISSUE THREAD

**Issue by Parth7223** [avniproject/apfodishanutrition#214]
**Changes to be done in Pre-release**

The following changes are to be made as part of this card.

- [x] Data source for column 'Breast feed till 2 years' needs to be updated. (**Data source** should be changed from Growth monitoring form to Child enrollment form)

**Comment by adamsanadi6** [comment]
@Parth7223 As discussed, this [[APFODISHA] Child linelist (AWC) (updated)](https://reporting-superset.avniproject.org/superset/dashboard/75) is an existing report currently in production and all necessary changes have been implemented, except for the encounter type name (Previously, it was "Child Follow Up" now it's "Growth Monitoring").

I have created a new dashboard report on prerelease [[APFODISHA Prerelease] Child linelist (AWC) (updated)](https://reporting-superset.avniproject.org/superset/dashboard/96) for better QA. Once QA is completed, we can implement the necessary changes to the existing report(Since we are having Row level security on the existing dataset) and delete the new one.
